app/views/users/profile_edit/_donations_by_causes.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "users/profile_sections/profile_section", title: "DONATIONS BY CAUSES"  do %>
+  <div class="donations-bars">
+    <% if @donations_by_causes.present? %>
+      <div class="left-arrow" >
+        <%= fa_icon "arrow-circle-left" %>
+      </div>
+
+      <% h_cause, h_percent = @donations_by_causes[0] %>
+      <%= render "users/profile_edit/donation_bar", cause: h_cause, percent: h_percent, highest: true %>
+
```

Comment:
I feel like lines 8-9 and 15-16 should be helper methods that will better explain what exactly is being done here.

something like `render_first_donation` and `render_rest_of_donations` something in that level to abstract and have better comprehension to what is being done in the view

Reply:
I can look into this.